src/components/UserManagement/UserTableData.jsx

```diff
@@ -42,7 +50,7 @@ const UserTableData = React.memo(props => {
       </td>
       <td>
         <a href={`/userprofile/${props.user._id}`}>{props.user.firstName}</a>
-      </td>
+      </td>i
```

Comment:
```suggestion
      </td>
```
This looks like  typo.